src/wp-admin/site-editor.php

```diff
@@ -87,6 +92,37 @@ static function( $classes ) {
 	'/wp/v2/global-styles/' . $active_global_styles_id . '?context=edit',
 	'/wp/v2/global-styles/' . $active_global_styles_id,
 	'/wp/v2/global-styles/themes/' . $active_theme,
+	array( $navigation_rest_route, 'OPTIONS' ),
+	array(
+		add_query_arg(
+			array(
+				'context'   => 'edit',
+				'per_page'  => 100,
+				'order'     => 'desc',
+				'orderby'   => 'date',
+				'_locale'   => 'user',
```

Comment:
The javascript should be refactored so this rest api call is not made twice.

Reply:
Agreed. We're tracking it in https://github.com/WordPress/gutenberg/issues/51803